charts/kubevault-metrics/templates/vaultserver/ops-kubevault-com-vaultopsrequest.yaml

```diff
@@ -0,0 +1,45 @@
+apiVersion: metrics.appscode.com/v1alpha1
+kind: MetricsConfiguration
+metadata:
+  name: ops-kubevault-com-vaultopsrequest
+spec:
+  targetRef:
+    apiVersion: ops.kubevault.com/v1alpha1
+    kind: VaultOpsRequest
+  commonLabels:
+    - key: app
+      valuePath: .spec.vaultRef.name
+    - key: app_namespace
+      valuePath: .metadata.namespace
+    - key: type
+      valuePath: .spec.type
+  metrics:
```

Comment:
Add some more metrics for VaultOpsRequest.
- created
- info (type can be added here as well)